### DISCUSSION TITLE
Looking for Journal Article of Terje Aven (Aven, 2015)

### DISCUSSION THREAD

**Discussion by JRBliekendaal** [JRBliekendaal/master-thesis#19] [Q&A] [answered]
@edzob 

Do you have a copy of the journal article on the subject of Risk Management by Terje Aven? A lot of research papers and essays are referring to (Aven, 2015). By the number of references, I feel obliged to evaluate the article for use, but the report is not public at this moment (I like open science more and more btw).

**Comment by edzob** [comment, marked as answer]
I thought I had retrieved it via citationsy but was not successful to
reproduce it.
I attached the pdf document.


Aven, T. (2015).
The concept of antifragility and its implications for the practice of risk
analysis.
Risk Analysis, 35(3):476–483.
Retrieved from:
https://dx.doi.org/10.1111/risa.12279,
https://www.researchgate.net/publication/266149936,
https://www.semanticscholar.org/paper/7488d14eacdb998ede95238573936a9b499112fb,

https://onlinelibrary.wiley.com/doi/abs/10.1111/risa.12279,
https://citationsy.com/archives/q?doi=10.1111/RISA.12279.


--
"It is amazing what you can accomplish if you do not care who gets the
credit." *-* Harry S Truman
*Signal*, Whatsapp, Telegram -
*+31629485425*
*Twitter*, Wire, Matrix Element - ***@***.****
*Threma* - *6FVWM3Z5*
*Homepage* - https://www.edzob.com/page/about
*ORCID* - *https://orcid.org/0000-0003-0097-7375
<https://orcid.org/0000-0003-0097-7375>*

**Reply by JRBliekendaal** [reply]
@edzob I cannot find the attachment. Are you sure you attached it?

**Reply by JRBliekendaal** [reply]
I discovered PaperPanda and this little chrome extension did find the document I was looking for.. So I found the source :D No action needed!

**Comment by edzob** [comment]
i think that sending attachment via email is not yet supported 😊

sci hub also did have the paper  🙃

Edzo

-- 
"It is amazing what you can accomplish if you do not care who gets the
credit." - Harry S Truman

Mobile, Signal, WhatsApp -
+31629485425
Twitter, riot.im, wire, Matrix -
@edzob

Homepage - https://www.edzob.com/page/about
ORCID - https://orcid.org/0000-0003-0097-7375

**Reply by JRBliekendaal** [reply]
found it with PaperPanda on sci hub 👯
